selftests/deployment/README.rst

```diff
@@ -0,0 +1,105 @@
+README
+======
+
+Here is a basic playbook (`deployment.yml`) to test the deployment of
+Avocado in a number of different environments, and installed via
+different methods.
+
+This code is intended to be used as regular playbook, so roughly the
+following steps should be followed:
+
+ 1) Install ansible. If you need help, refer to
+    https://docs.ansible.com/ansible/latest/installation_guide
+
+ 2) Adjust your inventory, that is, the hosts that will be used on the
+    playbook execution. The simplest way is to edit the `inventory`
+    file this directory.
+
+ 3) (OPTIONAL) adjust your variables on `vars.yml` or change the -e option via
+    command-line. Example::
+
+    $ ansible-playbook .... -e "method=pip"
+ 
```

Comment:
There are some unnecessary white spaces in some locations such as here.

Reply:
Nice catch. Fixed on v3.